UI/window-basic-main-outputs.cpp

```diff
@@ -1185,18 +1213,25 @@ inline void AdvancedOutput::SetupRecording()
 		obs_encoder_set_scaled_size(h264Recording, cx, cy);
 		obs_encoder_set_video(h264Recording, obs_get_video());
 		obs_output_set_video_encoder(fileOutput, h264Recording);
+		if (replayBuffer)
+			obs_output_set_video_encoder(replayBuffer, h264Recording);
 	}
 
 	for (int i = 0; i < MAX_AUDIO_MIXES; i++) {
 		if ((tracks & (1<<i)) != 0) {
 			obs_output_set_audio_encoder(fileOutput, aacTrack[i],
-					idx++);
+					idx);
+			if (replayBuffer)
+				obs_output_set_audio_encoder(replayBuffer, aacTrack[i], idx);
```

Comment:
Style: The line length limit is a strongly preferred 80 columns. This should be broken into two lines and properly indented.

Reply:
This is 78 columns in my editor. My tabs are rendered using 4 spaces. Are you assuming tabs are rendered using 8 spaces?

Reply:
From [the coding guidelines](https://github.com/jp9000/obs-studio/blob/master/CONTRIBUTING.rst#coding-guidelines):

> Tabs for indentation, spaces for alignment. Tabs are treated as 8 columns wide.

Reply:
Missed that, thanks.